jsonpickle/backend.py

```diff
@@ -98,6 +94,9 @@ def __init__(self, fallthrough=True):
 
         self.load_backend('simplejson')
         self.load_backend('json')
+        self.load_backend(
```

Comment:
If I recall correctly, aren't backends attempted in the order in which they are loaded? Might make sense to load ujson first, at least until people start using the yaml backend.

Reply:
True, sorry for not catching this until post-merge. I'll fix this up locally and push it out shortly.